devbin/model_extractor/Makefile

```diff
@@ -15,6 +15,7 @@ USE_BOOST=1
 USE_BOOST_FILESYSTEM_LIBS=0
 USE_BOOST_PROGRAMOPTIONS_LIBS=0
 # TODO
```

Comment:
Should this `TODO` be adjacent to `USE_RAPIDJSON`, not `USE_EIGEN`?

Reply:
I'm not sure, to be honest, I'll remove the comments and redundant definitions.